### PR TITLE
fixed example of cpu woq accuracy issue

### PR DESCRIPTION
**Pull request by PenghuiCheng**
## Type of Change

bug fix
No API changed

## Description

fixed example of cpu woq accuracy issue
## Expected Behavior & Potential Risk

Run the example of CPU WOQsuccessfully  with correct accuracy

## How has this PR been tested?

Local tested

### CODE REVIEW

File: intel_extension_for_transformers/transformers/modeling/modeling_auto.py [hunk 1]
```diff
@@ -342,42 +286,45 @@ def from_pretrained(cls, pretrained_model_name_or_path, *model_args, **kwargs):
...
-                        **kwargs,
-                    )
-                    model.config.update({"low_cpu_mem_usage": True})
-                except NotImplementedError:
-                    logger.info("Failed to load models with `low_cpu_mem_usage` specified, "
-                                "will fall to traditional load method with higher memory consumption.")
-                    kwargs["low_cpu_mem_usage"] = False
+                if use_xpu:
+                    kwargs["low_cpu_mem_usage"] = True
+                    kwargs["device_map"] = "auto"
```
Comment (kevinintel) [inline intel_extension_for_transformers/transformers/modeling/modeling_auto.py:291]: please find the real root cause

Comment (PenghuiCheng) [reply intel_extension_for_transformers/transformers/modeling/modeling_auto.py:291]: it is gptj issue, if we set low_cpu_mem_usage"=True, then the ACC is incorrect.

**Review by kevinintel — APPROVED** [review]

**Review by a32543254 — APPROVED** [review]
LGTM

File: intel_extension_for_transformers/llm/quantization/utils.py [hunk 2]
```diff
@@ -114,7 +114,7 @@ def _replace_linear(
                 with init_empty_weights():
                     in_features = module.in_features
                     out_features = module.out_features
-                    if device == "cpu" or device == torch.device("cpu"):
+                    if device == "cpu" or device == torch.device("cpu") or device == "auto":
```
Comment (hshen14) [inline intel_extension_for_transformers/llm/quantization/utils.py:117]: why need to add auto here?

**Comment by hshen14** [comment]
@PenghuiCheng  @VincyZhang  please add a test to avoid such regression in the future.

**Pull request merged**